protocol/src/main/java/io/camunda/zeebe/protocol/record/value/ProcessInstanceModificationRecordValue.java

```diff
@@ -33,7 +33,8 @@
   /** Returns a list of activate instructions (if available), or an empty list. */
   List<ProcessInstanceModificationActivateInstructionValue> getActivateInstructions();
 
-  Set<Long> getActivatedElementInstanceKeys();
+  @Deprecated
```

Comment:
:wrench: For a `@Deprecated` annotation, we should mention which method should be used instead.